modules/core/src/main/scala/org/scalasteward/core/edit/EditAlg.scala

```diff
@@ -33,12 +34,14 @@ final class EditAlg[F[_]](
     fileAlg: FileAlg[F],
     logger: Logger[F],
     sbtAlg: SbtAlg[F],
+    scalafmtAlg: ScalafmtAlg[F],
     workspaceAlg: WorkspaceAlg[F],
     F: Sync[F]
 ) {
   def applyUpdate(repo: Repo, update: Update): F[Unit] =
     for {
       _ <- applyScalafixMigrations(repo, update)
+      _ <- applyScalafmtConfUpdate(repo, update)
```

Comment:
I'm wondering since there is no `org.scalameta:scalafmt` artifact if we should just skip `applyScalafixMigrations` and `applyUpdateTo` for scalafmt updates. By doing this, we'd ensure that only `.scalafmt.conf` is edited. What do you think?

Reply:
Yeah, that sounds reasonable.
Fixed in https://github.com/fthomas/scala-steward/pull/719/commits/8274417bac6e194c07d4126b7aa5a61b7203138f
We no longer see `No files found that contain the current version` messages.